frontend/js/src/user/Listens.tsx

```diff
@@ -995,6 +1015,13 @@ export default class Listens extends React.Component<
                         this.handlePinnedRecording(pinnedListen)
                       }
                     />
+                    <MbidMappingModal
```

Comment:
No test for this, I'm not sure how to do this, or if we have detailed tests for other modals. happy to change this based on feedback

Reply:
added to LB-1179